packages/graphql/src/types/Suggestion.ts

```diff
@@ -7,6 +7,7 @@ export const SuggestionAction = objectType({
   name: 'SuggestionAction',
   definition(t) {
     t.nonNull.string('message');
+    t.string('priority');
```

Comment:
We should probably use an enum here as well https://nexusjs.org/docs/api/enum-type

Reply:
Good point, I updated to use this.